.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
   - "iojs"
-  - "8.x" 
-  - "7.x"
-  - "6.x"
-  - "5.x"
+  - "8"
```

Comment:
https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Even-more-Nodejs-versions seems to imply that "8" will only run the latest 8 such as 8.8.1, but not 8.1, 8.2, 8.3... 8.7 etc. Is everyone ok with that?